docs/mkdocs/docs/tutorials/data_organisation.md

```diff
@@ -0,0 +1,233 @@
+
+System problems are often caused by suspected bad data. It is worth thinking about how to investigate and remediate such problems. For example
+
+* Tooling to find and examine suspected bad data
+* Consider how to reconcile suspected bad data vs the original data source
+* Manual overrides to fix bad data to allow the system to run effectively while data is re-sourced
+
+### Maintenance
+
+Some routine maintenance is required to keep systems running efficiently and deal with problems, some of which may have causes from outside the system.
+
+For ArcticDB specifically, the following issues are worth consideration:
+
+* Version management (see [versions](#versions))
+* Defragmentation
+* Replication / Backup of data
+
+There are [Enterprise Features](#enterprise-features) to help with these issues and more.
+
+## General Performance Guidelines for ArcticDB
+
+In this section we want to highlight some general rules of thumb that will help you get the best performance from ArticDB.
+
+### Properties of ArcticDB
+
+It is worth understanding and bearing in mind these properties of the system
+
+#### Indices
+
+* The primary and performant index is the first DataFrame row index.
+* The columns names are also a performant index.
+* There are no secondary row or column indices.
+* The symbol-list is a good index when compacted.
+* The version-list is a good index when compacted.
+
+#### Data
+
+* DataFrame data is always stored columnar and compressed. ArcticDB is optimised to read and write these.
+* DataFrame data is tiled and stored in objects (default is 100,000 rows x 127 cols)
+* Data in different DataFrames is stored in different objects
+
+### ArcticDB is Optimised for Large DataFrames
+
+* Prefer a smaller number of symbols that have a large amount of data each
+* Accessing date ranges and subsets of columns is very efficient
+* Collecting together data from a large number of symbols can be slower
+
+### Plan your Version Management
+
+* In particular plan which versions to keep and how to delete the others
+* Snapshots are useful for version management
+* There are interactions between snapshots and versions: snapshots keep versions from being deleted
+
+Please see the [snapshots documentation](snapshots.md) and [snapshots notebook](../notebooks/ArcticDB_demo_snapshots.ipynb) for more details.
+
+
+### Lots of Small Appends or Updates Can Fragment Your Data
+
+Append and Update are efficient because they always add new chunks of data rather than reorganising the existing data (actually update will reorganise but only where necessary which is typically only a small amount).
+
+This means that lots of small appends or updates can result in lots of small data chunks, which makes reads slower.
+
+You can defragment a symbol manually using the [defragment_symbol_data](../api/library.md) library function. Altnernatively the [Enterprise Features](#enterprise-features) offer background processes that will take care of defragmentation for you.

```

Comment:
Altnernatively typo
Also, the other background processes mentioned so far already exist in Man Group, whereas defrag doesn't, and it isn't immediately obvious how to do it safely.